flink-libraries/flink-table/src/main/scala/org/apache/flink/table/descriptors/StreamTableDescriptor.scala

```diff
@@ -96,6 +96,6 @@ class StreamTableDescriptor(
 
     // this performs only basic validation
     // more validation can only happen within a factory
-    new StreamTableDescriptorValidator().validate(properties)
+    new StreamTableDescriptorValidator(true, true, true).validate(properties)
```

Comment:
Does it make sense to perform this validation here? This validation most probably has to be always repeated in the `*TableFactory`. Or am I missing something?

Reply:
I think we can also remove it. As the comment suggests it doesn't check much.